docs/resources/Stage_Instance.md

```diff
@@ -6,12 +6,21 @@ A _Stage Instance_ holds information about a live stage.
 
 ###### Stage Instance Structure
 
-| Field      | Type      | Description                                        |
-| ---------- | --------- | -------------------------------------------------- |
-| id         | snowflake | The id of this Stage instance                      |
-| guild_id   | snowflake | The guild id of the associated Stage channel       |
-| channel_id | snowflake | The id of the associated Stage channel             |
-| topic      | string    | The topic of the Stage instance (1-120 characters) |
+| Field                 | Type      | Description                                                                                                   |
+| --------------------- | --------- | ------------------------------------------------------------------------------------------------------------- |
+| id                    | snowflake | The id of this Stage instance                                                                                 |
+| guild_id              | snowflake | The guild id of the associated Stage channel                                                                  |
+| channel_id            | snowflake | The id of the associated Stage channel                                                                        |
+| topic                 | string    | The topic of the Stage instance (1-120 characters)                                                            |
+| privacy_level         | integer   | The [privacy level](#DOCS_RESOURCES_STAGE_INSTANCE/stage-instance-object-privacy-level) of the Stage instance |
+| discoverable_disabled | boolean   | Whether or not Stage discovery is disabled                                                                    |
+
+###### Privacy Level
+
+| Level      | Value | Description                                                         |
+| ---------- | ----- | ------------------------------------------------------------------- |
+| PUBLIC     | 0     | The Stage instance is visible publicly, such as on Stage discovery. |
+| GUILD_ONLY | 1     | The Stage instance is visible to only guild members.                |
```

Comment:
You missed them by 1.
1 is public.
2 is guild_only.

Reply:
My brain.